deploy/vagrant/bin/spot_request.py

```diff
@@ -203,12 +203,16 @@ def main(args):
         info('getting instance IPs...')
         for r in requests:
             instance_id = r.instance_id
+            info('waiting for ip to be allocated to the machine')
             ip = conn.get_only_instances([instance_id])[0].ip_address
```

Comment:
Yeah, we have to wait for the ip to be assigned because we have to wait for the spot request to be satisfied, it the user feels the waiting time is too long or anything goes wrong, he/she can just ctrl + c.